lib/persisted-model.js

```diff
@@ -483,6 +495,19 @@ module.exports = function(registry) {
   };
 
   /**
+   * Replace set of attributes.  Performs validation before replacing.
```

Comment:
`Replace set of attributes` sounds a bit vague to me. How about 

```
Replace all data of a model instance identified by the given ID
```

or perhaps

```
Replace all attributes of a model instance identified by the given ID
```